worksheet_D/oxo.py

```diff
@@ -0,0 +1,222 @@
+class OxoBoard:
+    def __init__(self):
+        """
+        For nxn I would have to choose a different
+        naming scheme something like ZeroZero,OneZero representing x and y
+        """
+
+            # I use the dictionary so I can have names next to my values for ease of changing them
+        self.fields = {"Top Left": 0, "Top Middle": 0, "Top Right": 0,
+                       "Middle Left": 0, "Middle Middle": 0, "Middle Right": 0,
+                       "Bottom Left": 0, "Bottom Middle": 0, "Bottom Right": 0}
```

Comment:
This data structure makes life difficult, as you constantly need to convert between (x,y) coordinates and names.

Note that in Python, dictionary keys do not have to be strings. For example you could use tuples of integers:
```
self.fields = { (0,0) : 0, (1,0): 0, (2,0): 0, ... }
```